source/client/services.yml

```diff
@@ -76,13 +76,62 @@ paths:
         '429':
           $ref: '../shared/standard-responses.yml#/responses/rateLimitedResponse'
 
+  services-id-edge-routers:
+    parameters:
+      - $ref: '../shared/parameters.yml#/id'
+    get:
+      summary: List of edge routers that a service can be dialed/bound for the current identity
```

Comment:
again, sounds awkward
`... that current identity can use to dial/bind given service`?